muswarmadmin/__main__.py

```diff
@@ -12,25 +14,38 @@
 loop = asyncio.get_event_loop()
 
 
-def pollAccumulate(count=0):
+async def pollAccumulate(count=0, client=None):
     """
     Poll the database until it is ready to answer queries
     """
+    if client is None:
+        client = SPARQLClient(ENV['MU_SPARQL_ENDPOINT'],
+                              graph=IRI(ENV['MU_APPLICATION_GRAPH']))
     if count >= int(ENV['POLL_RETRIES']):
+        await client.close()
         return False
-    payload = {'query': 'select distinct ?c where {[] a ?c } LIMIT 1'}
-    url = ENV['MU_SPARQL_ENDPOINT']
     try:
-        requests.get(url, params=payload)
-    except requests.RequestException:
-        logger.warn('SPARQL endpoint not yet ready')
+        result = await client.query("""
+            ASK
+            FROM {{graph}}
```

Comment:
I don't even think you need to specify a graph at all actually.

Reply:
Also, if you remove it, you don't need to specify the argument graph in the SPARQLClient object creation.